staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -33,7 +33,7 @@ const (
 
 var (
 	// Use buckets ranging from 25 ms to ~2.5 seconds.
-	latencyBuckets       = prometheus.ExponentialBuckets(25000, 2.5, 5)
+	latencyBuckets       = prometheus.ExponentialBuckets(0.0025, 2.5, 5)
```

Comment:
should it be `0.025` ?

Reply:
Actually, I think 0.000025 seconds is 25 microseconds, since 1 millisecond is 1000 microseconds and 1 second is 1000 milliseconds. 

@jpbetz do we need microsecond granularity? Most of our latency metrics seem to use latency buckets of millisecond granularity (e.g. https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/metrics/metrics.go#L150-L157).

Reply:
Thanks @danielqsj , yes, I want 25ms, not 2.5ms.

@logicalhan No, we don't need or want microsecond granularity. We just want to fix this so it does millisecond granularity as originally intended.

Reply:
Okay but you may want to amend the PR description, I was confused by it since you specified the time unit at microsecond there.